data/repository/src/commonMain/kotlin/io/github/droidkaigi/feeder/data/FeedRepositoryImpl.kt

```diff
@@ -18,7 +18,7 @@ open class FeedRepositoryImpl(
                     emit(feedApi.fetch())
                 }
             ) { favorites, apiFeed ->
-                FeedContents(apiFeed.sortedBy { it.publishedAt }, favorites)
+                FeedContents(apiFeed.sortedBy { it.publishedAt }.reversed(), favorites)
```

Comment:
It looks work fine! But do you think about using sortedByDescending() instead of sortedBy() and reversed() ?

Reply:
That sounds better >< Thank you!